docs/wiki/01_STUDY_PARAMETERS_SETUP/02_Clinical_Instruments/03_instrument_install.md

```diff
@@ -0,0 +1,72 @@
+## Installation
+
+The steps for installing a LORIS instrument depend on the instrument format chosen (**PHP** or **LINST**) and the data storage type selected (**SQL** table or **JSON**). This document details different installation options for all types of instruments. 
+
+### OPTION 1: SQL Table Storage
+#### LINST Instruments - The Instrument Manager 
+**The Instrument Manager** module is available to users with the appropriate permissions. This module allows front-end users to upload LINST instruments into the web browser which will then be automatically installed. This module only functions with LINST instruments which are set to use a dedicated SQL table in the database. 
+
+Follow the steps below in order to use the Instrument Manager for your instrument.
+
+  1. Navigate to the Instrument Manager module under the **Admin** menu.
+  2. Navigate to the upload tab within the module
+  3. Choose browse and select the file you have saved from the Instrument Builder (the file must have a `.linst` extension)
+  4. Click "Install" to submit your instrument.
+
+After installing your instrument, you can load it in a timepoint to test it. For instructions on populating instruments navigate directly to the [Instruments section](TODO) under Getting Started continue following the documentation.
```

Comment:
```suggestion
After installing your instrument, you can load it in a timepoint to test it.
```

This is just confusing.

Reply:
fixed it. I had forgotten to point to the correct place